test/src/org/labkey/test/tests/targetedms/TargetedMSOutlierHeatmapTest.java

```diff
@@ -0,0 +1,139 @@
+package org.labkey.test.tests.targetedms;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.components.targetedms.OutlierHeatmapSummaryWebPart;
+import org.labkey.test.components.targetedms.QCPlotsWebPart;
+import org.labkey.test.pages.panoramapremium.ConfigureMetricsUIPage;
+import org.labkey.test.pages.targetedms.PanoramaDashboard;
+import org.labkey.test.util.PortalHelper;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({})
+@BaseWebDriverTest.ClassTimeout(minutes = 5)
+public class TargetedMSOutlierHeatmapTest extends TargetedMSTest
+{
+    private static final String OUTLIER_HEATMAP_TAB = "Outlier Heatmap Summary";
+
+    @BeforeClass
+    public static void setupProject()
+    {
+        TargetedMSOutlierHeatmapTest init = (TargetedMSOutlierHeatmapTest) getCurrentTest();
+        init.doSetup();
+    }
+
+    @Override
+    protected void doCleanup(boolean afterTest)
+    {
+        _containerHelper.deleteProject(getProjectName(), afterTest);
+    }
+
+    private void doSetup()
+    {
+        setupFolder(FolderType.QC);
+        importData(SProCoP_FILE);
+
+        PortalHelper _portalHelper = new PortalHelper(getDriver());
+        _portalHelper.enterAdminMode();
+        _portalHelper.deleteTab("Raw Data");
+        _portalHelper.deleteTab("Pareto Plot");
+        _portalHelper.deleteTab("Annotations");
+        _portalHelper.addTab(OUTLIER_HEATMAP_TAB);
+
+        _portalHelper.addWebPart("Outlier Heatmap Summary View");
+        _portalHelper.exitAdminMode();
+    }
+
+    @Test
+    public void testHeatMapColorAndValues() throws InterruptedException
```

Comment:
I don't think anything throws this exception here.
```suggestion
    public void testHeatMapColorAndValues()
```